cluster-autoscaler/simulator/utilization/info.go

```diff
@@ -59,6 +60,16 @@ func Calculate(nodeInfo *framework.NodeInfo, skipDaemonSetPods, skipMirrorPods b
 		return Info{GpuUtil: gpuUtil, ResourceName: gpuConfig.ResourceName, Utilization: gpuUtil}, err
 	}
 
+	if draEnabled && len(nodeInfo.LocalResourceSlices) > 0 {
+		// TODO(DRA): Figure out how to calculate Node utilization for DRA resources properly. The current logic is provisional, it should work well for Nodes with a single Pool of
+		// expensive Devices but is probably not flexible enough for other scenarios.
+		resourceName, highestUtil, err := drautils.HighestDynamicResourceUtilization(nodeInfo)
+		if err != nil {
```

Comment:
I think it makes sense, my worry is just that any error in DRA integration that passes through here would de-facto break scale-down. The mitigation in this case is not clear to me - disabling DRA flag seems like the most obvious one, but then we would de facto just fallback to cpu + mem either way.

Overall, I don't want to block on this, especially as the whole utilization of slices is very iffy and provisional. Let's revisit this in the future, once we have a better idea on how this will be handled.

Reply:
> I think it makes sense, my worry is just that any error in DRA integration that passes through here would de-facto break scale-down.

That would be true only for Nodes using DRA (i.e. exposing ResourceSlices), right? And there would be a mitigation as you mention. IMO this is fine, especially for the MVP. 

> Overall, I don't want to block on this, especially as the whole utilization of slices is very iffy and provisional. Let's revisit this in the future, once we have a better idea on how this will be handled.

Ack!